src/SFA.DAS.EmployerFinance.Jobs/Program.cs

```diff
@@ -14,49 +13,27 @@ namespace SFA.DAS.EmployerFinance.Jobs
 {
     public static class Program
     {
-        private const string AppSettingFilePath = "appsettings.json";
-
         public static async Task Main()
         {
             ServicePointManager.DefaultConnectionLimit = 50;
 
             using (var container = IoC.Initialize())
             {
                 var startup = container.GetInstance<IStartup>();
-
-                await startup.StartAsync();
-
-                var jobActivator = new StructureMapJobActivator(container);
-
                 var environmentService = container.GetInstance<IEnvironmentService>();
-
-                var environmentName = environmentService.IsCurrent(DasEnv.LOCAL)
-                    ? EnvironmentName.Development
-                    : EnvironmentName.Production;
-
-                var hostBuilder = new HostBuilder()
+                var environmentName = environmentService.IsCurrent(DasEnv.LOCAL) ? EnvironmentName.Development : EnvironmentName.Production;
+                var jobActivator = new StructureMapJobActivator(container);
+            
+                var host = new HostBuilder()
                     .UseEnvironment(environmentName)
-                    .ConfigureWebJobs(builder =>
-                    {
-                        builder.AddAzureStorageCoreServices()
-                               .AddTimers();
-                    })
-                    .ConfigureAppConfiguration(builder =>
-                    {
-                        builder.AddJsonFile(AppSettingFilePath);
-                    })
-                    .ConfigureLogging(builder => 
-                    { 
-                        builder.SetMinimumLevel(LogLevel.Debug)
-                               .AddNLog();
-                    })
-                    .ConfigureServices(collection =>
-                    {
-                        collection.AddSingleton<IJobActivator>(jobActivator);
-                    })
-                    .UseConsoleLifetime();
-
-                var host = hostBuilder.Build();
+                    .ConfigureWebJobs(b => b.AddAzureStorageCoreServices().AddTimers())
+                    .ConfigureAppConfiguration(b => b.AddJsonFile(environmentName).AddEnvironmentVariables())
```

Comment:
This line throws an exception `System.IO.FileNotFoundException: The configuration file 'Development' was not found and is not optional.`. I think this line should be:

```c#
.ConfigureAppConfiguration(b => b.AddJsonFile("appsettings.json").AddJsonFile($"appsettings.{environmentName}.json", true))
```